<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDataqt5</class>
 <widget class="QDialog" name="SecDataqt5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image {
url(:/images/second.png)
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget {
border-image: url(:/images/second.png)
}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hello Mrs./Ms.</string>
    </property>
   </widget>
   <widget class="QLabel" name="loggedusername">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="display_from_database">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Display From Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="loggedID">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_image">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_image_Browse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>301</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Image_From_Database">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>441</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>220</number>
    </attribute>
    <column>
     <property name="text">
      <string>Image Name</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
